orangecontrib/timeseries/widgets/owtabletotimeseries.py

```diff
@@ -26,100 +57,255 @@ class Inputs:
     class Outputs:
         time_series = Output("Time series", Timeseries)
 
+    class Error(widget.OWWidget.Error):
+        invalid_time = widget.Msg("Invalid time")
+        year_out_of_range = widget.Msg("{}.")
+
+    class Warning(widget.OWWidget.Warning):
+        nan_times = widget.Msg("Rows with missing values of '{}' are skipped")
+
     want_main_area = False
     resizing_enabled = False
 
-    # Old settings that can't be migrated, but can be supported
-    selected_attr = settings.Setting('')
-    radio_sequential = settings.Setting(2)
-
-    settingsHandler = DomainContextHandler()
-    implied_sequence = settings.ContextSetting(0)
-    order = settings.ContextSetting(None)
-    autocommit = settings.Setting(True)
+    implied_sequence = Setting(0)
```

Comment:
Context setting can't be migrated into non-context because migrations are done with two separate (class) methods. As I recall, I've once done this; the trick was to *not* migrate the setting, but keep it in context settings and store it to non-context after calling `openContext`. But this obviously works only if you keep some context settings, and only migrate some of them to non-context.

In this case, I got rid of context settings altogether (because all settings in the bottom part would be non-context, so it is better to simply keep the only "context" setting, the variable name, as "hint" - the practice we already have in many widgets), so the above trick can't work. I don't see any way to migrate this.